modules/000-prom-test/openapi/openapi-case-tests.yaml

```diff
@@ -0,0 +1,16 @@
+positive:
+  values: []
+
+negative:
+  helmValues:
```

Comment:
You should delete this test example and add tests to the actual module.

Reply:
Done!